src/yb/yql/pgwrapper/pg_wrapper.cc

```diff
@@ -120,6 +120,14 @@ void ReadCommaSeparatedValues(const string& src, vector<string>* lines) {
   lines->insert(lines->end(), new_lines.begin(), new_lines.end());
 }
 
+void MergeSharedPreloadLibraries(const string& src, vector<string>* defaults) {
+  string copy = boost::erase_first_copy(boost::replace_all_copy(src, " ", ""), "shared_preload_libraries=");
```

Comment:
nit: long line.
Please wrap so that each line is 100 chars long or shorter.

Reply:
Done in https://github.com/yugabyte/yugabyte-db/pull/9576/commits/dc53f3c3b497f605e6dd7a97a2a03b12ed581d37.